pyproject.toml

```
@@ -308,7 +308,7 @@ url = "https://test.pypi.org/simple/"
 dace = {git = "https://github.com/GridTools/dace", tag = "__gt4py-next-integration_2025_08_06"}
```

DaCe version used in icon4py needs to follow the dace version used in the gt4py release.

```suggestion
dace = {git = "https://github.com/GridTools/dace", tag = "__gt4py-next-integration_2025_08_28"}
```

---

The correct dace version is not automatically installed because it is not a gt4py dependency, since there is no dace package for that tag. The dace version is installed in gt4py as part of a uv dev-group.